### ISSUE TITLE
Bot Run Assignment Failed

### ISSUE THREAD

**Issue by IsaDC** [w3c/aria-at-app#1226]
#### Steps to Reproduce:
1. Navigate to the Test Queue and select the "Navigation Menu" button (V24.09.19).
2. For either NVDA or VoiceOver, press the "Assign Testers" button and select the bot.

#### Expected Behavior:
The bot should be assigned to the selected test run.

#### Actual Behavior:
After the "Assign Testers" dialog is displayed, the bot is not assigned to any test run.

#### Alternate Procedure:
Tried adding the test plan from the "Data Management" page and selecting "Add Bot Run" for both NVDA and VoiceOver. This method also fails.

#### Additional Information:
Both bots run successfully in the sandbox environment, indicating that the issue seems specific to the test queue.

#### Suggested Solution:
Fixing this issue as soon as possible would be greatly appreciated, as it significantly speeds up the testing process.

**Comment by IsaDC** [comment]
### Update:
The bots are now being assigned correctly; however, the test run is immediately canceled after assignment.

**Comment by howard-e** [comment]
This should now be addressed in the [latest release](https://github.com/w3c/aria-at-app/blob/releases/CHANGELOG.md#190-2024-10-09)

cc @mcking65 @IsaDC @ccanash
